benchmarks-website/code.js

```diff
@@ -3,9 +3,14 @@ window.initAndRender = (function () {
     function stringToColor(str) {
         // Random colours are generally pretty disgusting...
         const MAP = {
-            "arrow": '#58067e',
-            "parquet": '#ef7f1d',
-            "vortex-file-compressed": '#23d100',
+            "df:arrow": '#7a27b1',
```

Comment:
it's a bit verbose, but better to spell out the full engine (since this page is actually public / we link to it from docs and website). So e.g., "datafusion:vortex", "duckdb:vortex", etc.

Reply:
sure